components/post_view/post_info/post_info.jsx

```diff
@@ -307,7 +307,7 @@ export default class PostInfo extends React.PureComponent {
                     delayShow={Constants.OVERLAY_TIME_DELAY}
                     placement='top'
                     overlay={
-                        <Tooltip>
+                        <Tooltip id='additional-information-overlay'>
```

Comment:
What happens if there are multiple posts with a tooltip?

Reply:
Good question! I believe the `id` prop does get used as an `id` HTML attribute, which needs to be unique.

Should we add a new required prop to the `PostInfo` component for this value?

Reply:
Could we generate a unique id using e.g. the post id?